.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
+  "prettier.singleQuote": true,
+  "prettier.jsxSingleQuote": true
+}
```

Comment:
I don't think it's a good idea to add this file to the codebase. This only configures prettier for VSCode, but is not usable by anyone who doesn't use VSCode, or a CI check.

How about we separate the task of adding Prettier to the checks into a separate PR?